README.md

```diff
@@ -765,7 +767,7 @@ make -j$( nproc ) install
 Your environment is set up. Now you can <a href="#runanode">build EOS and run a node</a>.
 
 <a name="manualdepubuntu"></a>
-### Clean install Ubuntu 16.04 & Higher
+### Clean install Ubuntu 16.04 & higher or Linux Mint 18 & higher
```

Comment:
Is the "& higher" part actually true?  There have been several reports of build failures in Ubuntu 17.

Reply:
Valid question. Im currently testing on Ubuntu 16.04. Your call, should I remove the "& Higher" reference for Ubuntu?

Reply:
I'm inclined to remove both bits of "& higher" verbiage until we have confirmation, especially with Ubuntu 18 (the next LTS release) on the horizon.

Reply:
Agreed I removed all references to "& Higher" for Ubuntu and Mint.